### PR TITLE
my portfolio project @Microverce  and complete mobile first app.

### PR DESCRIPTION
**Pull request by eerapheal**
This is my first Microverce project and my first mobile-first website.

### CODE REVIEW

**Review by Abenezer-Tilahun — COMMENTED** [review]
Hi @eerapheal,

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that has no pair programming. when I see the commit history your partner is not committing So, contribute to the project by both person.

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise.

File: style.css [hunk 1]
```diff
@@ -626,7 +626,7 @@ body {
 
 .form-btn:hover {
   background-color: #fff;
-  border: 1px solid rgb(248, 180, 8);
+  border: 1px solid yellow;
   color: #091e42;
```
Comment (codepantha) [inline style.css]: - [ ] Please implement the hover and the pressed states to be consistent with the Figma design shown below 👇 
- For hover, you need to change the `color`, `background-color` and `border-color`
- While for pressed, you only need to change the `background-color`
### Figma Design
image
### Your Design
image

**Review by codepantha — CHANGES_REQUESTED** [review]
# STATUS: REQUIRES CHANGES ♻️ 

## Hi @eerapheal 👋, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!



## Highlights 🎯 
✔️ Form fully integrated with Formspree and submitting successfully
✔️ Correct GitHub flow
✔️ No linter error

## Required Changes ♻️
_Check the comments under the review._

- [ ] Please write a more descriptive PR description that briefly describes the changes you have introduced into the codebase through this pull request 👍 

- [ ] Since this is a collaborative project, kindly include the name of your coding partner in the "Authors" section of your README 🙏 
- [ ] In your README, kindly license your project with the MIT license as shown in this [template](https://github.com/microverseinc/readme-template)

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- n/a

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by ichala — CHANGES_REQUESTED** [review]
Hi @eerapheal , 

Good job so far! 👏👏👏 
There are some issues that you still need to work on to go to the next project but you're almost there!

the-simpsons-homer-simpson

## To Highlight 🎉

- Great Design ✅ 
- Correct workflow ✅ 

 ## Required Changes ♻️

- [ ] as the previous reviewer requested , kindly add your teammates who participated in this project you the authors section of your readme file 💯 

## Optional suggestions

N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by GzimAsani — APPROVED** [review]
# STATUS: APPROVED :100:         
## There is nothing else to say than it's time to merge... :partying_face: 
Cheers and Happy coding! :clap:  :computer:  :wine_glass: 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


## Optional suggestions
-N/A

_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- N/A

### Please be reminded to not open a new pull request after you get a review, only use the same one :thought_balloon: 


 **_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**
 
 _As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by eerapheal** [comment]
Thanks for the review.

**Pull request merged**